testing/fusee-nano/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: TBK <alpine@jjtc.eu>
+# Maintainer: TBK <alpine@jjtc.eu>
+pkgname=fusee-nano
+pkgver=0.4
+pkgrel=0
+pkgdesc="A minimalist re-implementation of the Fusée Gelée exploit, designed to run on embedded Linux devices."
+url="https://github.com/DavidBuchanan314/fusee-nano"
+arch="all"
+license="MIT"
+options="!check" # No test suite
+makedepends="linux-headers"
+subpackages="$pkgname-udev::noarch"
+source="$pkgname-$pkgver.tar.gz::https://github.com/DavidBuchanan314/fusee-nano/archive/$pkgver.tar.gz"
+
+build() {
+    make
+}
+
+package() {
+    mkdir -p "$pkgdir"/usr/share/$pkgname/
+    cp -r files/* "$pkgdir"/usr/share/$pkgname/
+    install -Dm 755 fusee-nano "$pkgdir"/usr/bin/fusee-nano
+
+    # Remove openwrt file
+    rm "$pkgdir"/usr/share/$pkgname/20-tegra_rcm
+}
+
+udev() {
+    pkgdesc="udev rule for $pkgname"
+    depends="$pkgname=$pkgver-r$pkgrel eudev"
```

Comment:
This should be an [`install_if=`][install_if] directive, not a depends (in order to automatically get this subpackage when eudev is installed.

[install_if]:https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#install_if

Reply:
Done :)